installtests/test_helper.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+set -x 
+
+# Helper script to run 3 bash steps locally
```

Comment:
@oleg-nenashev  Yes, this helper script is admittedly a horrible, horrible hack because I was trying to get a docker container to start and run multiple commands in series and just couldn't get any of the 5 sensible approaches to work. :-(